kubecluster.yaml

```diff
@@ -5,6 +5,14 @@ description: >
   (as specified by the number_of_minions parameter, which defaults to 2).
 
 parameters:
+  plugin_version:
+    type: string
+    description: wlm plugin ver. Used cluster list view is shown.
```

Comment:
I think this is not correct. It indicates the template version for later update operations.